web/source/dom/domEventHandlers.ts

```diff
@@ -584,8 +584,8 @@ namespace com.keyman.dom {
     }.bind(this);
 
     // Also handles initial touch responses.
-    setFocusWithTouch(tEvent: {clientX: number, clientY: number, target?: EventTarget}) {
-      var touchX=tEvent.clientX,touchY=tEvent.clientY;
+    setFocusWithTouch(tEvent: {pageX: number, pageY: number, target?: EventTarget}) {
+      var touchX=tEvent.pageX,touchY=tEvent.pageY;
```

Comment:
The `getAbsoluteY` function operates from a `pageY` perspective.

This change is needed to properly handle caret positioning when the page is scrolled; my iPhone SE tests (see test spec) fail otherwise.

Reply:
Note the second line - there's an actual functional change here.

This link may help explain the distinction:  https://stackoverflow.com/a/21452887

Reply:
Ahh, I fell into the TypeScript trap again :smile: The event actually contains both `client*` and `page*` fields but we were using the wrong set. Thanks!